django/cantusdb_project/main_app/templates/chant_detail.html

```diff
@@ -267,10 +267,10 @@ <h4>List of melodies</h4>
                                 </select>
 
                                 {% if previous_folio %}
-                                    <a href="{% url "chant-list" %}?source={{ source.id }}&folio={{ previous_folio }}">{{ previous_folio }} <</a>
+                                    <a href="{% url "chant-list" source.id %}?folio={{ previous_folio }}">{{ previous_folio }} <</a>
                                 {% endif %}
                                 {% if next_folio %}
-                                    &nbsp;<a href="{% url "chant-list" %}?source={{ source.id }}&folio={{ next_folio }}">> {{ next_folio }}</a>
+                                    &nbsp;<a href="{% url "chant-list" source.id %}?folio={{ next_folio }}">> {{ next_folio }}</a>
```

Comment:
You can use keyword arguments in the `url` template tag so that you don't need to do string concatenation here...